asf_tools/flood_map.py

```diff
@@ -68,11 +77,15 @@ def __call__(self, **kwargs):
     return best_water_level
 
 
-def logstat(data, func=np.nanstd):
-    """ stat=logstat(data, func=np.nanstd)
-       calculates the statistic after taking the log and returns the statistic in linear scale.
-       INF values inside the data array are set to nan.
-       The func has to be able to handle nan values.
+def logstat(data: np.array, func=np.nanstd):
```

Comment:
```suggestion
def logstat(data: np.ndarray, func: Callable = np.nanstd) -> Union[np.ndarray, float]:
```

Might need to add this too:
```python
from typing import Callable
```